packages/generator/tests/acceptance/test_client_generator.py

```diff
@@ -72,6 +74,45 @@ def test___command_line_args___create_client_for_all_registered_measurements___r
     )
 
 
+@patch("pathlib.Path.cwd")
```

Comment:
This project uses [pytest-mock](https://pypi.org/project/pytest-mock/). Don't use patch directly.

Reply:
I have avoided patch, as this is acceptance test.